assets/css/src/components-public/_public.scss

```diff
@@ -89,6 +89,12 @@ $form-columns-space-between: 20px;
   .mailpoet_form_columns .mailpoet_form_column .mailpoet_paragraph:last-child {
     margin-bottom: 0;
   }
+
+  @include breakpoint-max-width($form-break-small-width) {
```

Comment:
I think it would be better instead of adding this new rule to modify the one above to set 0 bottom margin only for displays bigger than mobile. Because now we have 3 rules for the bottom margin:
1) adds margin
2) removes margin
3) add margin again
and every rule has a higher specificity. So I guess 2 rules would be nicer for future modifications.